websocket/client_test.go

```diff
@@ -1,67 +1,80 @@
 package websocket
 
 import (
+	"fmt"
+	"log"
 	"net"
 	"net/http"
 	"net/url"
+	"sync"
 	"testing"
 	"time"
 
 	"github.com/gorilla/websocket"
 )
 
 func TestWSClient(t *testing.T) {
+	var wg sync.WaitGroup
+	wg.Add(1)
+
 	http.HandleFunc("/test/client", func(w http.ResponseWriter, r *http.Request) {
+		defer func() {
+			fmt.Println("wg.Done")
```

Comment:
it is not a good style to use fmt and log in the test directly, you can use t.Log or t.Logf instead.

Reply:
3ks, i will fix it.